frontend/src/queries/nodes/DataTable/DataTableExport.tsx

```diff
@@ -223,6 +224,22 @@ export function DataTableExport({ query, fileNameForExport }: DataTableExportPro
                                 })
                             },
                         },
+                        {
+                            label: 'Save to a static cohort',
+                            onClick: () => {
+                                LemonDialog.open({
+                                    title: 'Save to a static cohort',
```

Comment:
The distinction between the two options is so subtle that I was confused the first time I tested this:

<img width="287" height="228" alt="Screenshot 2025-10-16 at 5 23 09 PM" src="https://github.com/user-attachments/assets/29b2ac0a-3a0b-40d2-a48b-c50e83970a98" />

Perhaps this should be: "Add to existing cohort" to make it more clear that this is adding to an existing cohort and is appending to it, not creating a new one. The fact that the cohort must be static is implied by the context (and the fact that it will only list static cohorts).

```suggestion
                            label: 'Add to existing cohort',
                            onClick: () => {
                                LemonDialog.open({
                                    title: 'Add to existing cohort',
```